File: src/main/java/io/github/allianaab2m/romechat/RomeChat.java
package io.github.allianaab2m.romechat;

import net.minecraftforge.eventbus.api.IEventBus;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;

@Mod(RomeChat.MOD_ID)
public class RomeChat {
    public static final String MOD_ID = "romechat";

    public RomeChat() {
        IEventBus modEventBus = FMLJavaModLoadingContext.get().getModEventBus();
    }
}

File: src/main/kotlin/io/github/allianaab2m/romechat/RomeChat.kt
package io.github.allianaab2m.romechat

import net.minecraftforge.fml.common.Mod
import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext

@Mod(RomeChat.MOD_ID)
class RomeChat {
    init {
        val modEventBus = FMLJavaModLoadingContext.get().modEventBus
    }

    companion object {
        const val MOD_ID: String = "romechat"
    }
}
